critiquebrainz/frontend/external/bookbrainz_db/edition_group.py

```diff
@@ -92,3 +92,47 @@ def fetch_multiple_edition_groups(bbids: List[uuid.UUID]) -> dict:
     if not results:
         return {}
     return results
+
+
+def fetch_work_for_edition_group(bbid: uuid.UUID):
```

Comment:
we should have a test for this method. I'm not sure if it makes sense to create our own custom relationship like we do in some other tests so that we know that they exist and won't change, or if we should just use an existing relationship. I think that if we choose a book that was published some time ago (e.g. 30-40 years) then the chances of a new edition coming out and breaking the tests is quite low, so probably an acceptable tradeoff.

Reply:
does this method return more than one work? if so, `fetch_works_for_edition_group`

Reply:
I've added the tests.